examples/mxnet_federated/README.md

```diff
@@ -0,0 +1,66 @@
+# MXNet: Basic Example
```

Comment:
Should we call this 
```suggestion
# MXNet: From Centralized To Federated
```
to be consistent with the PyTorch version?

Reply:
Which means we should also rename the directory to `mxnet_from_centralized_to_federated`